### DISCUSSION TITLE
New Addons for 1.19.2

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#784] [Compatibility Addon Suggestions]
With Dynamic Trees for 1.19 being released, we can begin working on 1.19 addons.
Following the same rules as the previous posts, comment under this post 1.19 mods that could have Dynamic Trees compatibility.
Suggestions from the [discord server](https://discord.gg/83qcpkY) will also be added here.

**Please please please include how many trees a mod adds when suggesting**

> DO NOT suggest the following types of mods as they will not be considered:
>  - Mods without a public source (such as MCreator mods)

Those in **bold** were considered and/or are being worked on.
When a mod is finished and published, it is removed from this list.

**Update from previous version:** (add-ons from previous versions that have yet to be updated)
- [Ars Nouveau](https://www.curseforge.com/minecraft/mc-mods/ars-nouveau)
- [Autumnity](https://www.curseforge.com/minecraft/mc-mods/autumnity)
- [Croptopia](https://www.curseforge.com/minecraft/mc-mods/croptopia)
- [Ecologics](https://www.curseforge.com/minecraft/mc-mods/ecologics)
- [Neapolitan](https://www.curseforge.com/minecraft/mc-mods/neapolitan)
- **[Quark](https://www.curseforge.com/minecraft/mc-mods/quark)**
- [Forbidden and Arcanus](https://www.curseforge.com/minecraft/mc-mods/forbidden-arcanus)
- [Pam's HarvestCraft 2: Trees](https://www.curseforge.com/minecraft/mc-mods/pams-harvestcraft-2-trees)
- [Traverse](https://www.curseforge.com/minecraft/mc-mods/traverse) (large scale)
- [Upgrade Aquatic](https://www.curseforge.com/minecraft/mc-mods/upgrade-aquatic)
- [The Aether](https://www.curseforge.com/minecraft/mc-mods/aether)
- [Integrated Dynamics](https://www.curseforge.com/minecraft/mc-mods/integrated-dynamics)
- [Tech Reborn](https://www.curseforge.com/minecraft/mc-mods/techreborn)
- [Twilight Forest](https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest) (large scale)
- [Hexerei](https://www.curseforge.com/minecraft/mc-mods/hexerei)
- [Project Vibrant Journeys](https://www.curseforge.com/minecraft/mc-mods/project-vibrant-journeys) (Large scale)
- [Tinker's Construct](https://www.curseforge.com/minecraft/mc-mods/tinkers-construct)

**New addons:**
- [Regions Unexplored](https://legacy.curseforge.com/minecraft/mc-mods/regions-unexplored) (large scale)
- [Pokecube AIO](https://www.curseforge.com/minecraft/mc-mods/pokecube-aoi)
- [Mystic's Biomes](https://www.curseforge.com/minecraft/mc-mods/mystics-biomes)

**Comment by moninformateur** [comment]
Definitely [Regions Unexplored](https://legacy.curseforge.com/minecraft/mc-mods/regions-unexplored)!

**Comment by KaykenJr** [comment]
terralith

**Comment by ZalikSprout** [comment]
A few I'd like to see added to 1.19.2 
[Hexerei] (3 trees)
[Project Vibrant Journeys](Multiple biomes)
[Tinker's Construct] 

I know I saw these one the 1.18.2 Addons and I would love to see them make their way over here too.

**Comment by ThisIsSP4RTAN** [comment]
[IC2 Classic](https://www.curseforge.com/minecraft/mc-mods/ic2-classic) (1.19.2) still just the rubber tree.

**Comment by ThisIsSP4RTAN** [comment]
Thermal Expansion. Pretty sure it's just the rubber wood tree

**Comment by PDelta41** [comment]
Ars Nouveau; hopefully it's a simple recompile.

**Comment by ebo2022** [comment]
I'd be interested in an addon for [William Wythers' Expanded Ecosphere](https://modrinth.com/mod/expanded-ecosphere), definitely large-scope and therefore cannot give a very accurate estimate on the tree count, but from a quick skim through the data looks like 150-200ish unique shapes/leave configurations

**Comment by Apocolyptik** [comment]
Would love to see an addon for [Productive Trees](https://www.curseforge.com/minecraft/mc-mods/productivetrees)! A large scale mod for sure since it focuses on tree varieties, but still would be lovely to see an addon for it

**Comment by JestemZbychu** [comment]
Is there any chance to ever see compability for [Immersive Weathering](https://www.curseforge.com/minecraft/mc-mods/immersive-weathering-forge)? Would be a killer match for DT!

**Comment by Late-Night** [comment]
sure hope we can see more update versions
and that regions unexplored one too
